bjoern/_bjoernmodule.c

```diff
@@ -10,9 +11,22 @@ run(PyObject* self, PyObject* args)
 
   PyObject* socket;
 
-  if(!PyArg_ParseTuple(args, "OO:server_run", &socket, &info.wsgi_app)) {
+#ifdef WANT_STATSD
+  StatsdInfo statsd_info;
```

Comment:
Maybe we also need a thread identifier in the statsd info since host + port aren't guaranteed to be unique